distribute_android/install-qt.sh

```diff
@@ -0,0 +1,339 @@
+#!/usr/bin/env bash
```

Comment:
Can you also use aqt to install qt? This script looks quite complicated to me.

Reply:
Don't know. I was really happy to have something working.
make it work, make it fast, make it clean...

Reply:
Strange. For me, it worked. Maybe the server is down? At least it says something like this in the CodeQl action at the moment.

Reply:
Yes. It seems to be down: https://forum.qt.io/topic/122837/download-qt-io-is-down